ansible/roles/debops.tinyproxy/defaults/main.yml

```diff
@@ -0,0 +1,501 @@
+# .. envvar:: tinyproxy__packages [[[
+#
+# List of APT packages which are required by the Tinyproxy application.
+tinyproxy__packages: []
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__allow [[[
+#
+# List of IP addresses or CIDR subnets which will be allowed to connect to the
+# Tinyproxy server in :command:`ip(6)tables` and TCP wrappers. If it's empty, remote
+# connections are not allowed.
+tinyproxy__allow: []
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__port [[[
+#
+# Port number on which this Tinyproxy listens on.
+tinyproxy__port: '8888'
+
+                                                                   # ]]]
+# Configuration for other Ansible roles [[[
+# -----------------------------------------
+
+# .. envvar:: tinyproxy__ferm__dependent_rules [[[
+#
+# Configuration for :ref:`debops.ferm` Tinyproxy role.
+tinyproxy__ferm__dependent_rules:
+
+  - type: 'accept'
+    dport: [ 'tinyproxy' ]
+    saddr: '{{ tinyproxy__allow }}'
+    accept_any: False
+    weight: '40'
+    role: 'tinyproxy'
+
+                                                                   # ]]]
+# .. envvar:: tinyproxy__etc_services__dependent_list [[[
+#
+# Configuration for the :ref:`debops.etc_services` Ansible role.
+tinyproxy__etc_services__dependent_list:
+
+  - name: 'tinyproxy'
+    port: '{{ tinyproxy__port }}'
+
+                                                                   # ]]]
+# Tinyproxy configuration file [[[
+# --------------------------------
+
+# The variables below define the contents of the
+# :file:`/etc/tinyproxy/tinyproxy.conf` configuration file.
+
+# .. envvar:: tinyproxy__default_configuration [[[
+#
+# The default configuration options which should be present in the main
+# configuration file.
+tinyproxy__default_configuration:
+
+  - name: 'tinyproxy.conf'
+    options:
+
+      - name: 'User Group'
+        comment: |
+          ##
+          ## tinyproxy.conf -- tinyproxy daemon configuration file
+          ##
+          ## This example tinyproxy.conf file contains example settings
+          ## with explanations in comments. For decriptions of all
+          ## parameters, see the tinproxy.conf(5) manual page.
+          ##
+
+
+          User/Group: This allows you to set the user and group that will be
+          used for tinyproxy after the initial binding to the port has been done
+          as the root user. Either the user or group name or the UID or GID
+          number may be used.
+        raw: |
+          User tinyproxy
+          Group tinyproxy
+        state: 'present'
+
+      - name: 'Port'
+        comment: |
+          Port: Specify the port which tinyproxy will listen on.  Please note
+          that should you choose to run on a port lower than 1024 you will need
+          to start tinyproxy using root.
+        raw: |
+          Port {{ tinyproxy__port }}
+        state: 'present'
```

Comment:
It looks like you use exclusively `raw` configuration scheme. It's fine of course, but there are many configuration options defined as `<name> <value>` pairs, which you could easily handle by specifying their value in the `value` parameter - that way these options could be easier to modify via inventory.

I suppose this could be implemented in the next iteration of the role, so no pressure about this one. :-)